koji/vars/external_repos.yml

```diff
@@ -8,40 +8,20 @@ external_repos:
   - name: epel-6
     url: file:///mnt/koji/external-repos/www/epel6-$arch/RPMS.epel/
   # EL7 repositories
-  - name: rhel-7.0-server
-    url: file:///mnt/koji/external-repos/rhel7-$arch/7.0/
   - name: epel-7
     url: https://dl.fedoraproject.org/pub/epel/7/$arch/
-  - name: centos-sclo-rh-rhel-7
-    url: http://mirror.centos.org/centos/7/sclo/$arch/rh/
-  - name: epel-7-beta
-    url: file:///mnt/koji/external-repos/www/epel7-$arch/RPMS.beta/
-  - name: rhscl-1.1-rhel-7
-    url: file:///mnt/koji/external-repos/rhscl-rhel7-$arch/1.1/
-  - name: rhel-7.0-server-optional
-    url: file:///mnt/koji/external-repos/rhel7-$arch/7.0-optional/
   - name: epel7
```

Comment:
Where is this repo used? In tool_belt it looks like we use `epel-7` everywhere.

Reply:
Both are currently used:

```
# koji list-external-repos --used --name=epel7
Tag                  Pri Mode       External repo name
-------------------- --- ---------- -------------------------
katello-4.0-rhel7-build 4   koji       epel7
katello-nightly-rhel7-build 10  koji       epel7
```

I think it was added to allow transition to or maybe just testing of using an external URL. We could probably update these two tags to `epel-7` and then update `epel-7` to use a URL.

Reply:
Perhaps @pcreech can enlighten us here.

Reply:
It was a change done initially to not clobber epel-7 with the http url.  Once we were comfortable, we also updated epel-7, but never switched katello back over from epel7.

Reply:
Excellent, I have cleaned up those two tags (seems tool_belt was already set properly) and will push an update that drops this.